### ISSUE TITLE
update_scm_breeze giving a git protocol error

### ISSUE THREAD

**Issue by AbdealiJK** [scmbreeze/scm_breeze#314]
I installed scm-breeze a few weeks ago.
Recently raised the bug https://github.com/scmbreeze/scm_breeze/issues/313 which I see has been solved in master,
So I thought - I need to upgrade to get it fixed for me.

I ran `update_scm_breeze` as was instructed in the README.
And I get this error:
```
$ update_scm_breeze
fatal: remote error:
  The unauthenticated git protocol on port 9418 is no longer supported.
Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
```

I am using Windows 10 + Ubuntu 18.04 WSL2 
I don't have issues with `git` in any of my other repos/projects

**Comment by ghthor** [comment]
If you look at the `type` for `update_scm_breeze` and read the source code for the function you'll see the steps that you need to take to update.

```shell
$ type update_scm_breeze 
update_scm_breeze is a function
update_scm_breeze () 
{ 
    currDir=$PWD;
    exec_scmb_expand_args builtin cd "$scmbDir";
    oldHEAD=$(git rev-parse HEAD 2> /dev/null);
    git pull origin master;
    source "$scmbDir/lib/scm_breeze.sh";
    _create_or_patch_scmbrc $oldHEAD;
    source "$scmbDir/scm_breeze.sh";
    exec_scmb_expand_args builtin cd "$currDir"
}
```

Do a quick check that the env variable is defined for scmbDir.

```shell
$ env | grep scmbDir
scmbDir=/home/ghthor/.scm_breeze
```

1. `cd "$scmbDir"` to scm_breeze directory
2. `git pull origin master`
3. `source "$scmbDir/lib/scm_breeze.sh"`
4. `source "$scmbDir/scm_breeze.sh"`

This step is unnecessary `_create_or_patch_scmbrc $oldHEAD`.

**Comment by ghthor** [comment]
Yes, the README should be updated as it's using `git://` good point.

**Comment by ghthor** [comment]
https://github.com/scmbreeze/scm_breeze/commit/62b73f7e78de9d73c72fc764bf28a025bdd9ec2d
